src/engine/TransitivePathBase.cpp

```diff
@@ -258,11 +258,8 @@ std::string TransitivePathBase::getCacheKeyImpl() const {
 // _____________________________________________________________________________
 std::string TransitivePathBase::getDescriptor() const {
   std::ostringstream os;
-  os << "TransitivePath ";
-  // If not full transitive hull, show interval as [min, max].
-  if (minDist_ > 1 || maxDist_ < std::numeric_limits<size_t>::max()) {
-    os << "[" << minDist_ << ", " << maxDist_ << "] ";
-  }
+  os << "TransitivePath "
+     << "{" << minDist_ << ", " << maxDist_ << "} ";
```

Comment:
doesn't this lead to a very large literal being displayed in every descriptor (which could be avoided)
In that case please leave it.

Reply:
You're right. Topic for another day.